src/tools/pub/glossary.md

```diff
@@ -146,16 +146,34 @@ and C are transitive ones.
 
 ## Uploader
 
-An uploader of a package is someone who has administrative permissions
-for that package. They can not only upload new versions of a package,
-but also [add and remove other uploaders](/tools/pub/cmd/pub-uploader)
-for that package. The uploader of a package is often, but not necessarily, the
-same as the [author](/tools/pub/pubspec#authorauthors) of a package.
+Someone who has administrative permissions for a package.
+A package uploader can upload new versions of the package,
+and they can also [add and remove other uploaders](/tools/pub/cmd/pub-uploader)
+for that package. 
+
+If a package has a verified publisher,
+then all members of the publisher can upload the package.
 
 Anyone uploading a new package automatically becomes an uploader for
```

Comment:
I would consider removing this paragraph. It repeats the last part of the first paragraph in this section.